Core/src/SplineInterpretation/SplineInterpretation.cpp

```diff
@@ -771,16 +768,19 @@ SplineInterpretationElement OpenInfraPlatform::Core::SplineInterpretation::Splin
 }
 
 SplineInterpretationElement OpenInfraPlatform::Core::SplineInterpretation::SplineInterpretation::obtainClothoid(
-	const SplineInterpretationElement previousElement,
-	const SplineInterpretationElement nextElement,
+	const SplineInterpretationElement& previousElement,
+	const SplineInterpretationElement& nextElement,
 	const carve::geom::vector<3>& startPoint,
-	SplineInterpretationElement& element) const throw(...) 
+	SplineInterpretationElement& element) const noexcept(true)
 {
 	element.setType("clothoid");
 	element.setStartpoint(startPoint);
 
 	// radius of clothoid (for calculation)
-	double radius;
+	//   0 is just a preset value;
+	//   the actual one will be obtained surely in one of the following if-cases,
+	//   because of the requirement of one arc at one clothoid end
+	double radius = 0;

```

Comment:
```suggestion
	double radius = 0.;
```

Initialize floating variables with floating numbers. Always.

Reply:
Done in commit e4fc298.